autobuild/configs/trtt.base.j2

```diff
@@ -0,0 +1,34 @@
+{
+    "TRTT": {
+        "Title": "Trittium",
+        "Address": "",
+        "Ip": "127.0.0.1",
+        "rpcPort": "{{ rpcPort|default(31002)}}",
+        "p2pPort": "{{ p2pPort|default(31001)}}",
+        "Username": "{{ rpcusername }}",
+        "Password": "{{ rpcpassword }}",
+        "AddressPrefix": "65",
+        "ScriptPrefix": "19",
+        "SecretPrefix": "239",
+        "COIN": "100000000",
+        "MinimumAmount": "0",
+        "DustAmount": "0",
+        "CreateTxMethod": "BTC",
+        "GetNewKeySupported": "true",
+        "ImportWithNoScanSupported": "true",
+        "MinTxFee": "0",
+        "BlockTime": "60",
+        "TxVersion": "1",
+        "FeePerByte": "25",
+        "Confirmations": "0",
+        "versions": {
+            "v4.0.2.1": {
+                "legacy": false,
+                "deprecatedrpc": false,
+                "xbridge_conf": "trittium--v4.0.2.1.conf",
+                "wallet_conf": "trittium--v3.0.2.1.conf",
```

Comment:
>                 "wallet_conf": "trittium--v3.0.2.1.conf",
Shouldn't this line be:
>                 "wallet_conf": "trittium--v4.0.2.1.conf",
?

Reply:
Yes. I have corrected it

Reply:
@8899kkA Did you push your commit with the correction? I'm not seeing the new version of `trtt.base.j2` in this PR.